tests/library/test_types.py

```diff
@@ -0,0 +1,18 @@
+from cellprofiler_library.types import sum_up_im 
+import numpy
+from pydantic import ValidationError
+
+def test_input_type_valid_throws_no_error():
+    try:
+        sum_up_im(numpy.ones((10, 10, 10)).astype(numpy.float32))
+        assert True
+    except Exception:
+        assert False
+
+
+def test_input_type_invalid_throws_error():
+    try:
+        sum_up_im(numpy.ones((10, 10)).astype(numpy.float32))
+        assert False
+    except ValidationError:
```

Comment:
Have to catch ValidationError. Pydantic seems to capture the custom `CellProfilerInputValidationError` and raise this one instead. 
the `@validate_call(...)` decorator in `types.py` raises the `pydantic.ValidationError`